modules/api/src/main/java/org/eclipse/xpanse/api/OrchestratorApi.java

```diff
@@ -412,12 +411,12 @@ public Response destroy(@PathVariable("id") String id) {
      */
     @Tag(name = "Service Vendor",
             description = "APIs to manage register services.")
-    @GetMapping(value = "/service/openapi/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @GetMapping(value = "/register/openapi/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
produces JSON -> but return type is string.

Reply:
I will remove the produces config.

Reply:
Is it possible to redirect the static pages of the API document?
My initial idea is to make this api as the endpoint of the APIs.

Reply:
I will redirect the url.